### DISCUSSION TITLE
bug when downloading files

### DISCUSSION THREAD

**Discussion by CharlesUk** [iterate-ch/cyberduck#15877] [Q&A]
Hi,
I'm using Cyberduck 8.8.2 on :
Édition	Windows 11 Famille
Version	23H2
Build du système d’exploitation	22631.3447
Windows Feature Experience Pack 1000.22688.1000.0

I use Cyberduck to manage the files of my websites.
Each time I try to save all the files locally on my computer I have an error message. 
I notice this bug on differents servers.

here is the screenshot :
![Capture d'écran 2024-04-24 152927](https://github.com/iterate-ch/cyberduck/assets/52503238/2e895473-9e9f-43f9-8277-53e6799c1945)

What does it mean ?
Can you help me ?
I love your sofware but I will use another if this problem persist.

**Comment by tatze96** [comment]
The error message indicates that the listing of the folder named `04` failed because the connection got interrupted within the process.

Please check if the server and network or, respectively, internet connection runs stable without issues.

**Reply by CharlesUk** [reply]
Hi,
I have already check with my hosting provider an everythings seems to be fine on his side.
For the internet connection, it is stabke, I didn't notice any slow down or disconnect. I have a good fiber connection.
That's why I don't undertand.
To help you I notice that's happens oftne when I try to download heavy folders or a full website in one shot.
If I download manually file by file, the error don't show.
